jmxtrans-core/pom.xml

```diff
@@ -134,6 +134,11 @@
 			<scope>system</scope>
 			<systemPath>${toolsjar}</systemPath>
 		</dependency>
+		<dependency><!-- quartz depends on Log4J -->
```

Comment:
Do we really need this dependency? Wouldn't having it in the jmxtrans module be sufficient? Or maybe as a test dependency if we want to log from log4j during test. Since jmxtrans-core isn't defining a runtime artifact (it has to be packaged in the jmxtrans module), it seems to me that defining the logging implementation should be done somewhere else.

Reply:
It's a not a logging implementation, it's a bridge:
```
quartz -> log4j-over-slf4j -> slf4j -> logback
```

Reply:
Yep, wrong naming on my side. Still, since we are not using log4j in this module, but log4j comes from transitive dependencies, it looks to me like a packaging concern more suited for the jmxtrans module. But I don't have a strong opinion on this. Feel free to merge.

Reply:
I see your point, I'll check if I can remove it.

Reply:
@gehel You were right log4j-over-slf4j was useless because quart uses slf4j api (and log4j at runtime). It's just needed by Kafka and Zookeeper servers in integration tests.